package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "desafio-instaq",
+  "version": "1.0.0",
+  "description": "## Primeira versão do readme do repo.",
+  "main": "dist/server.js",
+  "scripts": {
+    "start": "ts-node-dev src/server.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@types/express": "^5.0.0",
```

geralmente fixamos a versão das dependências para não correr riscos de quebrar a aplicação com atualizações
```suggestion
    "@types/express": "5.0.0",
```

---

Perfeito!, não sabia disso,  isso meio que faz com que a nossa aplicação não quebre com updates aleatórios né?, porque no momento que fixamos as dependência não corremos o risco de por acaso  do nada receber um update e quebrar o funcionamento das dependências, certo?